INSTRON/iocBoot/iocINSTRON-IOC-01/config.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" ?>
+<ioc_config xmlns:xi="http://www.w3.org/2001/XInclude">
+ <config_part>
+  <ioc_desc>Instron 8800 Stress Rig</ioc_desc>
+  <ioc_details>Instron 8800 Stress Rig</ioc_details>
+  <macros>
+    <macro name="LVDCOM_OPTIONS" pattern="^\d?\d?$" description="Options that define how the VI is started. Add selected options together to make the final value. 1 - warn if idle, 2 - start if idle, 4 - stop VIs if started on exit, 6 - stop VI on exit. (Default 1)" />
```

Comment:
Is stop VI on exit really 6? 8 would seem more logical in an exponential pattern